### PR TITLE
Add BME280 Sensor Component

### PR DESCRIPTION
**Pull request by brentru**
Adding support for the BME280 I2C or SPI Temperature Humidity Pressure Sensor (https://www.adafruit.com/product/2652)

### CODE REVIEW

File: components/i2c/bme280/definition.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+          "sensorType":"humidity",
+          "defaultPeriod":900
+       },
+       {
+          "displayName":"Pressure Sensor",
+          "sensorType":"pressure",
+          "defaultPeriod":900
+       },
+       {
+          "displayName":"Altitude Sensor",
```
Comment (ladyada) [inline components/i2c/bme280/definition.json:23]: if we add this we have to note that we assume the sea level pressure so its relative changes not absolute

**Review by ladyada — CHANGES_REQUESTED** [review]

File: components/i2c/bme280/definition.json (hunk 1 above)
Comment (brentru) [reply components/i2c/bme280/definition.json:23]: @ladyada In the future, we'll have an "advanced configuration" on the form for the user to add their location's sea level pressure... For now, are you OK with changing this `displayName` to "Altitude Sensor (Relative)", or omitting it from the form entirely?

Comment (ladyada) [reply components/i2c/bme280/definition.json:23]: yep thats a good solution!

Comment (brentru) [reply components/i2c/bme280/definition.json]: great, resolved in https://github.com/adafruit/Wippersnapper_Components/pull/12/commits/3494f6e1911d3bbc7096cffe3d43c546e9c34890

File: components/i2c/bme280/definition.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
+{
+    "displayName":"BME280",
+    "i2cAddresses":[
+       "0x76, 0x77"
```
Comment (lorennorman) [inline components/i2c/bme280/definition.json:4]: This is wrong, should be `"i2cAddresses": [ "0x76", "0x77" ],`

**Review by lorennorman — APPROVED** [review]
fix the i2c addresses and this is good to go.

File: components/i2c/bme280/definition.json (hunk 2 above)
Comment (brentru) [reply components/i2c/bme280/definition.json]: Resolved in https://github.com/adafruit/Wippersnapper_Components/pull/12/commits/d0b76a5364adf4c8f9fc9b32345b744424971d48

**Pull request merged**